specs/agents/logging.md

```diff
@@ -31,11 +30,134 @@ agents can treat that as a synonym for `error` or `fatal`.
 
 The `off` level is a switch to completely turn off logging.
 
-### Backwards compatibility
+## Backwards compatibility
 
 Most agents have already implemented `log_level`,
 accepting a different set of levels.
 Those agents should still accept their "native" log levels to preserve backwards compatibility.
 However, in central config,
 there will only be a dropdown with the levels that are consistent across agents.
 Also, the documentation should not mention the old log levels going forward.
+
+## Logging Preamble
+
```

Comment:
nit: It might be helpful to lead with the reasoning for the preamble -- which is, IIUC, to ensure that *by default*, logging out from agents includes information that is commonly useful for support cases.

Reply:
Good suggestion 👍 , I'll add that.